src/view/ThemeManager.js

```diff
@@ -167,7 +167,7 @@ define(function (require, exports, module) {
             filename: fixPath(theme.file._path)
         });
 
-        parser.parse("#editor-holder {" + content + "\n}", function (err, tree) {
+        parser.parse("#editor-holder, .force-syntax-highlighting {" + content + "\n}", function (err, tree) {
```

Comment:
Let's change `.force-syntax-highlighting` to something more generic because we are not enforcing that css rule to only apply to syntax highlighting, which make the name misleading.  Let's call it something more along the lines of `.theme-override`.